tables/intro-table/step4.md

```diff
@@ -1,4 +1,4 @@
-Even basic tables need to be able to account for the case of large dataset. In this step, we'll look at how we can address this from a styling perspective.
+A common case for Table is the need to handle large datasets elegantly. This can be addressed on both the styling and interactivity side. In this step, we'll look at how we can address the case of large datasets from a styling perspective.
```

Comment:
Typically, tables should be designed and built to handle large datasets elegantly, this means that considerations need to be taken in order to style them and build their interactivity....

Reply:
I'm not sure what the shortcoming is with the original paragraph, or how the suggestion improves it. Can you elaborate?

Reply:
I was attempting to merge the two sentences together, but feel free to just take it as a suggestion